src/canvas/PatternPath.tsx

```diff
@@ -38,19 +43,19 @@ export class PatternPath implements IPatternPath {
 
         // If the path2D only has 1 item in it and there are not more points to add to the path2D, 
         // then the path is valid and we can return it.
-        if (!this._lastIndexAddedToPath2D && this._lastIndexAddedToPath2D === this._points.length - 1) {
+        if (this._lastIndexAddedToPath2D === 0 && this._points.length === 1) {
```

Comment:
What does this change do?

Reply:
I kept bumping up on that line and struggling to understand what it meant. I thought it was easier to understand like that, since lastIndexAddedToPath2D isn't a boolean. It changes nothing, it's just expressed differently.

Reply:
Thanks! I read it more carefully. This looks okay to me.